### DISCUSSION TITLE
Does ISG (Intelligent Security Graphic) include recommended block rules?

### DISCUSSION THREAD

**Discussion by HotCakeX** [MicrosoftDocs/WDAC-Toolkit#216] [Q&A] [answered]
I've been looking for this information but couldn't find it anywhere.

I need to know when I use ISG policy rule option, do I still need to use and merge [Microsoft recommended driver block rules](https://learn.microsoft.com/en-us/windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-driver-block-rules) and [Microsoft recommended block rules](https://learn.microsoft.com/en-us/windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-block-rules) with my policy or are they already included and blocked by ISG authorization?

Thanks

**Comment by jgeurten** [comment, marked as answer]
ISG does not include the recommended blocklist, no.

**Reply by HotCakeX** [reply]
Thank you!
